### PR TITLE
add getPostSlugLink  and getPostSiteLink functions

### PR DESCRIPTION
**Pull request by sinirzi**
closes: #228 

Co-Authored-By: Rasit 12167194+durmusrasit@users.noreply.github.com
Co-Authored-By: ikadgzl <ikadgzl@gmail.com>

### CODE REVIEW

File: apps/pano/app/utils/index.ts [hunk 1]
```diff
@@ -93,14 +93,24 @@ export function validatePassword(password: unknown): password is string {
 
 export function getExternalPostURL(post: PostWithCommentCount) {
   const location = global.location;
-  const postUrl = `${location.origin}/posts/${post.slug}/${post.id}`;
+  const postUrl = `${location.origin + getPostSlugLink(post)}`;
```
Comment (usirin) [inline apps/pano/app/utils/index.ts:96]: if you are not gonna interpolate, please remove the backticks:

```suggestion
  const postUrl = location.origin + getPostSlugLink(post);
```

File: apps/pano/app/utils/index.ts [hunk 2]
```diff
@@ -93,14 +93,24 @@ export function validatePassword(password: unknown): password is string {
...
   return postUrl;
 }
 export function getExternalCommentURL(comment: Comment) {
   const location = global.location;
   const commentUrl = `${location.origin}${location.pathname}#c_${comment.id}`;
   return commentUrl;
 }
+
+export function getPostSlugLink(post: PostWithCommentCount) {
```
Comment (usirin) [inline apps/pano/app/utils/index.ts:105]:
```suggestion
export function getPostLink(post: PostWithCommentCount) {
```

File: apps/pano/app/utils/index.ts [hunk 3]
```diff
@@ -93,14 +93,24 @@ export function validatePassword(password: unknown): password is string {
...
+  const postUrl = `/posts/${post.slug}/${post.id}`;
+  return postUrl;
+}
+
+export function getPostSiteLink(post: PostWithCommentCount) {
```
Comment (usirin) [inline apps/pano/app/utils/index.ts:110]:
```suggestion
export function getSitePostsLink(post: PostWithCommentCount) {
```

**Review by usirin — APPROVED** [review]

**Comment by usirin** [comment]
@sinirzi can you please update the code with given suggestions?

**Comment by sinirzi** [comment]
> @sinirzi can you please update the code with given suggestions?

thank you for the suggestion. I changed accordingly

**Review by usirin — APPROVED** [review]

**Pull request merged**